packages/website/src/components/linter/createLinter.ts

```diff
@@ -159,7 +159,23 @@ export function createLinter(
       console.log('[Editor] Updating', fileName, compilerOptions);
       parser.updateConfig(compilerOptions);
     } catch (e) {
-      console.error(e);
+      const lintMessage: Linter.LintMessage = {
+        source: 'eslint',
+        ruleId: '',
+        severity: 2,
+        nodeType: '',
+        column: 1,
+        line: 1,
+        message: String(e instanceof Error ? e.message : e),
```

Comment:
You are reporting it as a lint error as if it's coming from our tool. It would be better described as coming from TypeScript and should be reported as if it's a type-checking error. It should also have proper position information if possible.

Reply:
Thanks for the review.
After revisiting it, I feel like I can't solve it(I really cant understand how the monaco editor works :sweat_smile:)
I'll close this pr.